core/find_apk.sh

```diff
@@ -8,7 +8,15 @@ getlatestapk() {
   tempapks=( )
   OLDIFS="$IFS"
   IFS="
-" # We set IFS to newline here so that spaces can survive the for loop
+" # We set IFS to newline here so that filenames containing spaces are
+  # handled correctly in the for-loops below.
+
+  # decompress apks
+  # some apks are lz compressed to workaround github file-size limits.
+  for foundapklz in $(find "$1" -iname '*.apk.lz'); do
+    lunzip $foundapklz
```

Comment:
You can check out the code at https://github.com/opengapps/opengapps/blob/master/download_sources.sh#L63-L69

it has the example commands for extraction and for clean-up